src/pymodaq_plugins_thorlabs/daq_move_plugins/daq_move_KPZ101.py

```diff
@@ -0,0 +1,109 @@
+from pymodaq.control_modules.move_utility_classes import DAQ_Move_base, main, comon_parameters_fun
+from pymodaq.utils.logger import set_logger, get_module_name
+
+from pymodaq_plugins_thorlabs.hardware.kinesis import serialnumbers_piezo, Piezo
+
+
+logger = set_logger(get_module_name(__file__))
+
+
+class DAQ_Move_KPZ101(DAQ_Move_base):
+    """
+    Wrapper object to access Piezo functionalities, similar to Kinesis instruments 
+    """
+    _controller_units = 'V'
+    _epsilon = 0.0
+
+    is_multiaxes = False
+
+    stage_names = []
+
+    params = [{'title': 'Controller ID:', 'name': 'controller_id', 'type': 'str', 'value': '', 'readonly': True},
+              {'title': 'Serial number:', 'name': 'serial_number', 'type': 'list',
+               'limits': serialnumbers_piezo},
+              ] + comon_parameters_fun(is_multiaxes, epsilon=_epsilon)
+
+    def ini_attributes(self):
+        self.controller: Piezo = None
+        self.settings.child('bounds', 'is_bounds').setValue(True)
+        self.settings.child('bounds', 'max_bound').setValue(360)
+        self.settings.child('bounds', 'min_bound').setValue(0)
+
+    def commit_settings(self, param):
+       pass 
+
+    def ini_stage(self, controller=None):
+        """
+        Connect to Kinesis Piezo Stage by communicating with kinesis.py
+        """
+        self.controller = self.ini_stage_init(controller, Piezo())
+
+        if self.settings['multiaxes', 'multi_status'] == "Master":
```

Comment:
you could use::

self.is_master

Reply:
We updated the self.controller to include self.is_master format.